main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"encoding/json"
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+)
+
+type Items struct {
+	XMLName xml.Name `xml:"items" json:"-"`
+	Items   []Item   `xml:"item" json:"items"`
+}
+
+type Item struct {
+	XMLName     xml.Name `xml:"item" json:"-"`
+	URL         string   `xml:"url" json:"url"`
+	PhoneNumber string   `xml:"phone_number" json:"phone_number"`
+	Location    string   `xml:"location" json:"location"`
+	Menu        []Menu   `xml:"menu" json:"menu"`
+}
+
+type Menu struct {
+	XMLName xml.Name `xml:"menu" json:"-"`
+	Value   []Value  `xml:"value" json:"value"`
+}
+
+type Value struct {
+	XMLName  xml.Name `xml:"value" json:"-"`
+	MenuName string   `xml:"menu_name" json:"menu_name"`
+	Price    int      `xml:"price" json:"price"`
+}
+
+func ParseXML() Items {
+	xmlFile, err := os.Open("test.xml")
+	if err != nil {
+		fmt.Println("error opening file.")
+	}
+	fmt.Println("Opened...")
```

Comment:
Opened what? Once again, it doesn't give me any information about the context around the log entry